frontend/src/citizen-frontend/income-statements/IncomeStatementView.tsx

```diff
@@ -0,0 +1,362 @@
+          }
+        />
+      )}
+      {entrepreneur.limitedCompany && (
+        <Row
+          light
+          label={t.income.view.limitedCompany}
+          value={
+            entrepreneur.limitedCompany.incomeSource === 'INCOMES_REGISTER'
+              ? t.income.view.limitedCompanyIncomesRegister
+              : t.income.view.limitedCompanyAttachments
+          }
+        />
+      )}
+      {entrepreneur.partnership && (
+        <Row
+          light
+          label={t.income.view.partnership}
+          value={t.income.view.attachments}
+        />
+      )}
+      {entrepreneur.lightEntrepreneur && (
+        <Row
+          light
+          label={t.income.view.lightEntrepreneur}
+          value={t.income.view.attachments}
+        />
+      )}
+      {entrepreneur.accountant && (
+        <AccountantInfo accountant={entrepreneur.accountant} />
+      )}
+    </>
+  )
+}
+
+function AccountantInfo({ accountant }: { accountant: Accountant }) {
+  const t = useTranslation()
+  return (
+    <>
+      <H3>{t.income.view.accountantTitle}</H3>
+      <Row label={t.income.view.accountant} value={accountant.name} />
+      <Row label={t.income.view.email} value={accountant.email} />
+      <Row label={t.income.view.phone} value={accountant.phone} />
+      <Row label={t.income.view.address} value={accountant.address} />
+    </>
+  )
+}
+
+function CitizenAttachments({ attachments }: { attachments: Attachment[] }) {
+  const t = useTranslation()
+  return (
+    <>
+      <H2>{t.income.view.citizenAttachments.title}</H2>
+      {attachments.length === 0 ? (
+        <p>{t.income.view.citizenAttachments.noAttachments}</p>
+      ) : (
+        <Row
+          label={`${t.income.view.attachments}:`}
+          value={<UploadedFiles files={attachments} />}
+        />
+      )}
+    </>
+  )
+}
+
+function UploadedFiles({ files }: { files: Attachment[] }) {
+  return (
+    <FixedSpaceColumn>
+      {files.map((file) => (
+        <div key={file.id}>
+          <FileIcon icon={fileIcon(file)} />
+          <FileDownloadButton
+            file={file}
+            fileFetchFn={getAttachmentBlob}
+            onFileUnavailable={() => undefined}
+          />
+        </div>
+      ))}
+    </FixedSpaceColumn>
+  )
+}
+
+const FileIcon = styled(FontAwesomeIcon)`
+  color: ${(p) => p.theme.colors.main.primary};
+  margin-right: ${defaultMargins.s};
+`
+
+function Row({
+  label,
+  light,
+  value
+}: {
+  label: string
+  light?: boolean
+  value: React.ReactNode
+}) {
+  return (
+    <>
+      <FixedSpaceRow>
+        <LabelColumn light={light}>{label}</LabelColumn>
+        <div>{value}</div>
+      </FixedSpaceRow>
+      <Gap size="s" />
+    </>
+  )
+}
+
+function makeYesNo(t: Translations) {
+  return (value: boolean): string => {
+    return value ? t.common.yes : t.common.no
+  }
+}
+
+const LabelColumn = styled(Label)<{ light?: boolean }>`
+  flex: 0 0 auto;
+  width: 250px;
+  ${(p) => (p.light ? 'font-weight: 400;' : '')}
```

Comment:
(Not for this PR specifically, mainly venting) We really ought to make variables for font weights, too, as we have very specific weights available/supported of specific fonts.